Source/WebKit/UIProcess/WebPageProxy.messages.in

```diff
@@ -87,6 +87,7 @@ messages -> WebPageProxy {
 
 #if ENABLE(INPUT_TYPE_COLOR)
     [EnabledBy=InputTypeColorEnabled] ShowColorPicker(WebCore::Color initialColor, WebCore::IntRect elementRect, Vector<WebCore::Color> suggestions);
+    [EnabledBy=InputTypeColorEnhancementsEnabled] ShowColorPickerEnhanced(WebCore::Color initialColor, WebCore::IntRect elementRect, bool supportsAlpha, Vector<WebCore::Color> suggestions);
```

Comment:
Is this separate IPC message necessary? (Maybe we could just `MESSAGE_CHECK` that `InputTypeColorEnhancementsEnabled` is set, if we ever get `SupportsAlpha::Yes`)

Reply:
This is what @pxlcoder suggested and I think @gavin-apple might also prefer? I would be okay with a `MESSAGE_CHECK` of sorts though.

Reply:
I've gone ahead with your suggestion @whsieh. Hopefully I am using the API correctly.